Parquet/src/main/java/io/deephaven/parquet/ParquetFileWriter.java

```diff
@@ -68,19 +73,19 @@ public RowGroupWriter addRowGroup(long size) {
     }
 
     public void close() throws IOException {
+        final OutputStream os = Channels.newOutputStream(writeChannel);
+        os.write(ParquetFileReader.MAGIC);
         serializeOffsetIndexes(offsetIndexes, blocks, writeChannel);
```

Comment:
Pass `os` into this method, too. It makes new ones, in a loop, and that's gross.